chef/data_bags/crowbar/migrate/manila/001_add_max_header_line.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade(ta, td, a, d)
+  unless a.has_key? "max_header_line"
+    a["max_header_line"] = ta["max_header_line"]
+  end
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
```

Comment:
Unused method argument - `td`. If it's necessary, use `_` or `_td` as an argument name to indicate that it won't be used.